docs/DEPLOYMENT.md

```
@@ -135,21 +135,31 @@ Ensure `.env` is never committed. `.env.example` remains tracked.
 2. Pull/build images:
 
 ```bash
-docker compose --env-file .env -f config/docker-compose.services.yaml build
+docker compose --env-file ~/.axon/.env -f docker-compose.yaml build
 ```
 
 3. Start the stack:
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 3; Style: 1/1/1

(MD029, ol-prefix)